daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-raw.md

```diff
@@ -0,0 +1,128 @@
+---
+type: docs
+title: "Pub/Sub without CloudEvents"
+linkTitle: "Pub/Sub without CloudEvents"
+weight: 6000
+description: "Use Pub/Sub without CloudEvents." 
+---
+
+## Introduction
+
+Dapr uses CloudEvents to provide additional context to the event payload, enabling features like:
+* Tracing
+* Deduplication by message Id
+* Content-type for proper deserialization of event's data
+
+For more information about CloudEvent, please check the [official specification](https://github.com/cloudevents/spec).
+
+When adopting Dapr to an application, it might happen that other publisher or subscriber apps do not use Dapr and still require to communicate via Pub/Sub. For this reason, since version 1.2, Dapr enables apps to publish and subscribe to events that are not wrapped in a CloudEvent.
```

Comment:
```suggestion
When adding Dapr to your application, some services may still need to communicate via raw pub/sub messages not encapsulated in CloudEvents. This may be for compatibility reasons, or because some apps are not using Dapr yet. Dapr enables apps to publish and subscribe to raw events that are not wrapped in a CloudEvent.
```